src/SDKs/ApplicationInsights/ApplicationInsights.Tests/ApplicationInsights.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <Description>ApplicationInsights.Tests Class library</Description>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="[1.1.3-preview,2.0.0)" />    
```

Comment:
Please use the latest version of Resource Manager nuget package, the latest I believe is 1.6.0

Reply:
@shahabhijeet it is a range, do we want to make the change. I saw other projects doing the same thing. 
And some project is using <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />

So should I use 1.6.0-preview or 1.6.0 directly? thanks!

Reply:
@shahabhijeet updated. thanks!